tuftool/Cargo.toml

```
@@ -36,7 +36,7 @@ snafu = { version = "0.6.10", features = ["backtraces-impl-backtrace-crate"] }
 structopt = "0.3"
 tempfile = "3.1.0"
 tokio = "~1.8"  # LTS
-tough = { version = "0.11.1", path = "../tough", features = ["http"] }
+tough = { version = "0.11.2", path = "../tough", features = ["http"] }
 tough-ssm = { version = "0.6.1", path = "../tough-ssm" }
 tough-kms = { version = "0.3.1", path = "../tough-kms" }
```

Shouldn't these be updated to the bumped versions as well?

```suggestion
tough-ssm = { version = "0.6.2", path = "../tough-ssm" }
tough-kms = { version = "0.3.2", path = "../tough-kms" }
```

---

Good catch - I'll update them!